test/onmt/ExtendedCmdLineTest.lua

```diff
@@ -100,8 +100,11 @@ end
 function cmdLineTest.fail_unknown()
   local cmd = onmt.utils.ExtendedCmdLine.new()
   onmt.Seq2Seq.declareOpts(cmd)
-
+  onmt.data.SampledDataset.declareOpts(cmd)
+  -- will generate error
+  print('')
```

Comment:
it is just for conveniency when applying test units - with the print, it shows:
```
146/211 ExtendedCmdLineTest_fail_unknown ................................ [WAIT]        
invalid argument: -xxx
invalid dependency for option -importance_sampling: requires '-sample' option
146/211 ExtendedCmdLineTest_fail_unknown ................................ [PASS]
``` 

otherwise, the first warning is on the [WAIT] line